core/src/test/java/org/bitcoinj/core/ECKeyTest.java

```diff
@@ -72,15 +70,18 @@ public void setUp() throws Exception {
     public void sValue() throws Exception {
         // Check that we never generate an S value that is larger than half the curve order. This avoids a malleability
         // issue that can allow someone to change a transaction [hash] without invalidating the signature.
-        final int ITERATIONS = 10;
```

Comment:
Why this change to `byte`? Feels weird to keep a count in a byte.

Reply:
Because the code that generates the hashes uses a single-byte buffer and this value shouldn't be made larger than a byte without some consideration.

Reply:
Ok, I see.

Ultimately as I understand the code it just feeds pseudo-random data into the signer.

But for the purpose of this refactoring, it's fine I guess.